ledger/ledger-runner-common/src/test/scala/com/daml/ledger/runner/common/PureConfigReaderWriterSpec.scala

```diff
@@ -356,6 +357,45 @@ class PureConfigReaderWriterSpec
       "type = unsafe-jwt-hmac-256\nsecret=mysecret2",
       AuthServiceConfig.UnsafeJwtHmac256("mysecret2"),
     )
+    compare(
+      "type = unsafe-jwt-hmac-256\nsecret=mysecret3",
+      AuthServiceConfig.UnsafeJwtHmac256("mysecret3", None),
+    )
+    compare(
+      "type = unsafe-jwt-hmac-256\nsecret=mysecret3\nleeway-options={leeway: 1}",
```

Comment:
this string becomes to long and I suggest to make use of multi-strings with """ ... """.stripMargin

Reply:
I would also suggest to stick to a single convention of hocon, which would be then smth like:

```
      """
        |type = unsafe-jwt-hmac-256
        |secret =m ysecret3
        |leeway-options {
        |  leeway = 1
        |}  
        |""".stripMargin,
```

Reply:
note btw you have leeway mentioned twice which is a bit confusing ;) I do not have a better solution though